Documentation/api.md

```diff
@@ -597,8 +597,10 @@ TLSConfig specifies TLS configuration parameters.
 | ----- | ----------- | ------ | -------- |
 | caFile | Path to the CA cert in the Prometheus container to use for the targets. | string | false |
 | caSecret | Secret containing the CA cert to use for the targets. | *[v1.SecretKeySelector](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/#secretkeyselector-v1-core) | false |
+| caConfigMap | ConfigMap containing the CA cert to use for the targets. | *v1.ConfigMapKeySelector | false |
```

Comment:
I think this should rather be one field, with two sub-fields where the user can choose to use a configmap key selector or secretkeyselector.